windows/windows_update_install/windows_update_install.yml

```diff
@@ -22,7 +22,7 @@
             skip_msg: >-
               Skip test case due to parameter windows_nfs_share or windows_nfs_msu_path is not provided 
               and no need to test the .msu file installation.
-            skip_reason: "Skipped"
+            skip_reason: "Not Applicable"
```

Comment:
"Not Applicable" means current VM config can not run current test case, here this test case is controlled by parameters set or unset, not VM config.

Reply:
Won't change it. Keep it as it is. Close the merge request. Thanks.